### ISSUE TITLE
Interested in take over Laravel Elasticsearch 

### ISSUE THREAD

**Issue by shuvroroy** [mailerlite/laravel-elasticsearch#149]
Hi @cviebrock 

I am Shuvro Roy. I am a Senior Software Engineer at [MailerLite](https://www.mailerlite.com). In **MailerLite** we use this package. So we are interested to take over this package & maintain it. So let me know how we can discuss more about it

**Comment by cviebrock** [comment]
😄 I've send an email to the address listed on your github profile page.

**Comment by PrinsFrank** [comment]
@shuvroroy @cviebrock This is one of the fastest takeovers of open source repositories I've seen, and it worries me. How are you going to ensure continuity @shuvroroy when the first thing you did was add as many references to MailerLite, including updated namespaces which is a breaking change? @cviebrock Why did you give over control of this package? Would you recommend users to migrate away from this?

**Comment by PrinsFrank** [comment]
@cviebrock I see you are still the sole maintainer of the package on packagist. Are you planning to relinquish control and transfer it to MailerLite? If not, why not archive the package and point to a fork so people can make a decision themselves if they want to switch forks?

**Comment by cviebrock** [comment]
@PrinsFrank thank you for your very valid concerns.

I have not really maintained the package in any meaningful sense for well over a year.  I had a notice on the README asking for any interested parties to contact me regarding taking over development.  @shuvroroy asked, and I responded.

In retrospect, transferring ownership of the repository was probably not the best way to handle this.  You are correct: Mailerlite could just have easily created a fork of the repo, made all their changes there, and published it themselves to Packagist.  That way users could leave their code the same until such time as they needed a newer version of the package (and then adjust namespaces, etc., etc.).

I have confidence, based on the number of people that have installed my package, that it will be maintained by Mailerlite and new features/bugfixes/etc. will be dealt with promptly ... at least faster than I could promise.  At the end of the day, though, if they don't maintain it, or make it so it doesn't serve the wider needs of the Laravel community ... I (respectfully) don't care.  That's the beauty of Opensource software development!  If you don't like what someone does, you can fork their code and go your own way with it (original license permitting, of course).

However, in order to make the transition for end-users more palatable, I've "reinstated" the original cviebrock/laravel-elasticsearch repository (so the auto-redirects that Github added are removed), but made it clear that the package is no longer maintained.  Users like yourself should either fork it or switch to mailerlite/laravel-elasticsearch at their convenience.  (There is a slight issue with how Packagist automagically handles Github ownership transfers, so I've asked them to make some corrections there as well ... might take a while).

In summary, I apologize for the temporary inconvenience of the transfer.  You should (shortly) be able to use the latest version of my (unmaintained/abandoned) package without any breaking changes until such time as you are ready to make a switch.

Thanks,

- Colin Viebrock

**Comment by nklmilojevic** [comment]
@PrinsFrank thanks for your concerns. We at MailerLite are a global (and remote) company, and we are a user of many open-source tools and libraries and try to contribute where we can. One of our core services is ElasticSearch and when we saw that the library will not be maintained and that @cviebrock doesn't have time to maintain it anymore, we reached out.

The first change we did is indeed changing the namespace because MailerLite has taken over the development of it. You are indeed right - it is a breaking change, and as such it will be released as a major version, per semver. 

I can ensure you that this isn't a "hostile" takeover and that maintainers and contributors are welcome to send new PRs and the team will review it and merge it asap. We have already forked the repo before and maintained it on our own, but it seemed logical to reach to Colin to transfer it under MailerLite.

I would just like to thank @cviebrock for making the library and to promise that we will keep maintaining it and listen to the users of it. 

I'm here if I need to clarify something - feel free to ask anything.

Thanks!
